Homework10/GenericArray.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Homework10
+{
+    public class GenericArray<T> where T : Human
+    {
+        private T[] _array;
+
+        public GenericArray(int size)
+        {
+            _array = new T[size + 1];
+        }
+
+        public int GetAraryLength() => _array.Length;
+
+        public void AddElement(int index, T element)
```

Comment:
Please make this method universal. It can throw IndexOutOfRangeException

Reply:
Ok, Get the point. Just create one more method for adding element to array. E.g. you have the array [T el1, T el2] and you want to add T el3